values.yaml

```diff
@@ -21,3 +21,18 @@ cloudflared:
   image:
     repository: "cloudflare/cloudflared"
     tag: "2024.12.2"
+  userTunnel:
+    enabled: false  
```

Comment:
Maybe we should drop a comment in here about Quick Tunnels as default, with a link to the Cloudflare docs.

Reply:
Done